agent/pkg/api/entry_streamer_socket_connector.go

```diff
@@ -22,7 +22,8 @@ func (e *DefaultEntryStreamerSocketConnector) SendEntry(socketId int, entry *tap
 	if params.EnableFullEntries {
 		message, _ = models.CreateFullEntryWebSocketMessage(entry)
 	} else {
-		extension := extensionsMap[entry.Protocol.Name]
+		protocol := protocolsMap[entry.ProtocolUniqueName]
+		extension := extensionsMap[protocol.Name]
```

Comment:
Why does `extensionsMap` use `protocolsMap[entry.ProtocolUniqueName].Name` and not `entry.ProtocolUniqueName`?

Reply:
so we won't have to duplicate values, http has several values in the protocols map (grpc, gqp...) but only one value in extentionsMap

Reply:
So there are many protocols with different `ProtocolUniqeName` but with the same `Name`?
That's unintuitive.